include/ck_tile/ops/gemm/pipeline/gemm_universal_pipeline_ag_bg_cr_skip_b_lds_policy.hpp

```diff
@@ -0,0 +1,334 @@
+// SPDX-License-Identifier: MIT
+// Copyright (c) 2025, Advanced Micro Devices, Inc. All rights reserved.
+
+#pragma once
+
+#include "ck_tile/core.hpp"
+#include "ck_tile/ops/gemm/warp/warp_gemm_dispatcher.hpp"
+#include "ck_tile/ops/common/tensor_layout.hpp"
+#include "ck_tile/ops/gemm/pipeline/gemm_universal_pipeline_ag_bg_cr_policy.hpp"
+namespace ck_tile {
+
+template <typename Derived>
+struct UniversalGemmSkipBLdsBasePolicy
```

Comment:
Can't you actually reuse here this class: https://github.com/ROCm/composable_kernel/blob/5f885d2b7af1e6b2f40eefa9126f58e93e164e6d/include/ck_tile/ops/gemm/pipeline/gemm_universal_pipeline_ag_bg_cr_policy.hpp#L13

You don't have to use all it's functionality.
Maybe you could even refactor it out to separate file?